cli/flox-rust-sdk/src/models/manifest/raw.rs

```diff
@@ -388,22 +388,37 @@ impl RawManifest {
         "#});
 
         // `systems` array with custom formatting
-        let mut systems_array = Array::new();
-        for system in systems {
-            let mut item = Value::from(system.to_string());
-            item.decor_mut().set_prefix("\n  "); // Indent each item with two spaces
-            if Some(system) == systems.last() {
-                item.decor_mut().set_suffix(",\n"); // Add a newline before the first item
+        if systems.len() != 4 {
```

Comment:
Can you do an unordered comparison against `lockfile::DEFAULT_SYSTEMS_STR` in case we change the default systems in the future?